docs/docs/consumer-development/changeover-procedure.md

```diff
@@ -167,7 +167,6 @@ Example of a consumer chain addition proposal.
     // Must be unique from all other consumer chain ids of the executing provider chain.
     "chain_id": "standalone-1",
     // Initial height of new consumer chain.
-    // For a completely new chain, this will be {0,1}.
```

Comment:
Why remove this?

Reply:
This was one of Lexas comments; since this is in the changeover-procedure.md, this might be confusing info.
This comment is still there in onboarding.md